src/py/mat3ra/made/tools/build/defect/builders.py

```diff
@@ -322,6 +325,69 @@ def create_adatom(
         return [self.merge_slab_and_defect(new_material, only_adatom_material)]
 
 
+class PointDefectPairBuilder(PointDefectBuilder):
+    _ConfigurationType: type(PointDefectPairConfiguration) = PointDefectPairConfiguration  # type: ignore
+    _GeneratedItemType: Material = Material
+
+    def _create_defect(self, defect_configuration: PointDefectConfiguration) -> Material:
```

Comment:
_create_point_defect?

Don't we have a user-facing helper function for this functionality already?